app/src/pages/instance/organizationsPage/organizationsPageHeader/organizationsExport/organizationsExport.tsx

```
@@ -46,8 +45,9 @@ export const OrganizationsExport = ({ appliedFiltersCount }: OrganizationsExport
   const query = useSelector(querySelector) as QueryParams;
 
   const handleClick = () => {
-    const filtersParams: QueryParams = { ...query, offset: 0, limit: ORGANIZATIONS_EXPORT_LIMIT };
+    const filtersParams: QueryParams = { ...query, offset: 0 };
     const data = prepareQueryFilters(filtersParams);
+    delete data.limit;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)